jsonrpc.go

```diff
@@ -205,7 +205,7 @@ type RPCError struct {
 
 // Error function is provided to be used as error object.
 func (e *RPCError) Error() string {
-	return strconv.Itoa(e.Code) + ":" + e.Message
+	return strconv.Itoa(e.Code) + ": " + e.Message
```

Comment:
Since this is an exposed string and some may have hard coded against such string, this should not be changed.